### ISSUE TITLE
Are failed payments retried?

### ISSUE THREAD

**Issue by JoeMurray** [iATSPayments/com.iatspayments.civicrm#372]
A client had a slew of recurring transactions not go through yesterday. Will they be retried automatically, or should they put through manual payments? Perhaps the answer could be put into the README.md. Thx.

**Comment by adixon** [comment]
Yes, there's documentation here: https://github.com/iATSPayments/com.iatspayments.civicrm/wiki/Recurring-Contribution-Failure-Handling

tldr; handling failures is inherently difficult, but most cases should be handled reasonably well without intervention, and you can generate reports to help administrators review and take action.

**Comment by adixon** [comment]
Yes, there's documentation here: https://github.com/iATSPayments/com.iatspayments.civicrm/wiki/Recurring-Contribution-Failure-Handling

tldr; handling failures is inherently difficult, but most cases should be handled reasonably well without intervention, and you can generate reports to help administrators review and take action.

**Comment by adixon** [comment]
I've updated the readme with a note and this link as well.
